pkg/git/gitlab/gitlab_client.go

```diff
@@ -136,6 +145,14 @@ func (g *GitlabClient) UndoPaCMergeRequest(repoUrl string, d *gp.MergeRequestDat
 			return "", err
 		}
 		d.BaseBranchName = baseBranchName
+	} else {
+		baseBranchExists, err := g.branchExist(projectPath, d.BaseBranchName)
```

Comment:
if base branch does not exist, it means nothing to clean up, so it should be no-op instead of an error.

Reply:
right fixed